src/main/java/seedu/address/logic/commands/UnassignCaptainCommand.java

```diff
@@ -18,11 +18,11 @@
  * <p>
  * Example usage:
  * <pre>
- * {@code stripCaptain p/Sergio Ramos}
+ * {@code unassignCaptain p/Sergio Ramos}
  * </pre>
  */
-public class StripCaptainCommand extends Command {
-    public static final String COMMAND_WORD = "stripcaptain";
+public class UnassignCaptainCommand extends Command {
+    public static final String COMMAND_WORD = "unassignCaptain";
```

Comment:
This should be unassigncaptain

```suggestion
    public static final String COMMAND_WORD = "unassigncaptain";
```